src/bionics.cpp

```diff
@@ -910,7 +910,7 @@ void player::process_bionic(int b)
 
     // Bionic effects on every turn they are active go here.
     if (bio.id == "bio_night") {
-        if (calendar::turn % 5) {
+        if (calendar::is_time_for(5)) {
```

Comment:
Btw. this inverts the logic: the message will be printed every 5 turns, currently it is printed on turn 1,2,3,4 but not on turn 5. But this is probably a good thing. Just noting this in case the original logic was somehow important.